src/pages/settings/Security/TwoFactorAuth/DisablePage.tsx

```diff
@@ -39,14 +39,14 @@ function DisablePage() {
                 <View style={[styles.ph5, styles.mt3]}>
                     <Text>{translate('twoFactorAuth.explainProcessToRemove')}</Text>
                 </View>
-            </ScrollView>
-            <FixedFooter style={[styles.mt2, styles.pt2]}>
-                <View style={[styles.mh5, styles.mb4]}>
+                <View style={[styles.mh5, styles.mb4, styles.mt5]}>
```

Comment:
I think we should apply the same margin style as [ValidateCodeActionModal](https://github.com/Expensify/App/blob/37de406bf312e2fa263b0f0199017c41ed287e6d/src/components/ValidateCodeActionModal/index.tsx#L93)
```suggestion
                <View style={[styles.mh5, styles.mb4, styles.mt3]}>
```